report/assets/styles.css

```diff
@@ -167,6 +167,9 @@
   --pass-icon-url: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><title>check</title><path fill="%23178239" d="M24 4C12.95 4 4 12.95 4 24c0 11.04 8.95 20 20 20 11.04 0 20-8.96 20-20 0-11.05-8.96-20-20-20zm-4 30L10 24l2.83-2.83L20 28.34l15.17-15.17L38 16 20 34z"/></svg>');
   --average-icon-url: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><title>info</title><path fill="%23E67700" d="M24 4C12.95 4 4 12.95 4 24s8.95 20 20 20 20-8.95 20-20S35.05 4 24 4zm2 30h-4V22h4v12zm0-16h-4v-4h4v4z"/></svg>');
   --fail-icon-url: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><title>warn</title><path fill="%23C7221F" d="M2 42h44L24 4 2 42zm24-6h-4v-4h4v4zm0-8h-4v-8h4v8z"/></svg>');
+  --error-icon-sm-url: url('data:image/svg+xml;utf8,<svg width="18" height="28" viewBox="0 0 18 28" xmlns="http://www.w3.org/2000/svg"><title>lighthouse error</title><path d="M7.5 22H10.5V19H7.5V22Z" fill="%23FF4E42"/><path d="M7.5 16H10.5V7H7.5V16Z" fill="%23FF4E42"/></svg>');
+  --error-icon-md-url: url('data:image/svg+xml;utf8,<svg width="36" height="36" viewBox="0 0 36 36" xmlns="http://www.w3.org/2000/svg"><title>lighthouse error</title><path d="M17 26H20V23H17V26Z" fill="%23FF4E42"/><path d="M17 20H20V11H17V20Z" fill="%23FF4E42"/></svg>');
+  --error-icon-lg-url: url('data:image/svg+xml;utf8,<svg width="120" height="120" viewBox="0 0 120 120" xmlns="http://www.w3.org/2000/svg"><title>lighthouse error</title><path d="M56 80H64V72.0001H56V80Z" fill="%23FF4E42"/><path d="M56 64H64V41H56V64Z" fill="%23FF4E42"/></svg>');
```

Comment:
svgs should scale just fine on their own, why do we need 3 different sizes?

Reply:
agreed, the svgs will scale fine on their own. We don't need the 3 different sizes so I'll go ahead and update this.